crates/macro/src/derive_props/builder.rs

```diff
@@ -36,12 +35,9 @@ impl ToTokens for PropsBuilder<'_> {
             ..
         } = props;
 
-        let step_trait_repeat = iter::repeat(step_trait);
-        let vis_repeat = iter::repeat(&vis);
-
         let build_step = self.build_step();
-        let impl_steps = self.impl_steps();
-        let set_fields = self.set_fields();
+        let impl_steps = self.impl_steps().into_iter();
```

Comment:
`impl_steps()` returns a `TokenStream` so looks like it was not supposed to be repeated, whoops! Can you revert this and instead change `#(#impl_steps)*` to `#impl_steps`